besu/src/main/java/org/hyperledger/besu/cli/BesuCommand.java

```diff
@@ -419,6 +420,13 @@ void setBannedNodeIds(final List<String> values) {
               + " NONE disables NAT functionality. (default: ${DEFAULT-VALUE})")
   private final NatMethod natMethod = DEFAULT_NAT_METHOD;
 
+  @SuppressWarnings({"FieldCanBeFinal", "FieldMayBeFinal"}) // PicoCLI requires non-final Strings.
+  @Option(
+      names = {"--Xnat-manager-pod-name"},
```

Comment:
Could we create a `KubernetesConfiguration` object, i suspect we will have more Kubernetes options in the future.

Reply:
Or maybe `NatConfiguration` to group all NAT related CLI options.

Reply:
Also i would not use the term `manager` in the name of the option. I would use something like `--Xnat-kubernetes-pod-name` or `--Xnat-kube-pod-name`

Reply:
> Or maybe `NatConfiguration` to group all NAT related CLI options.

I was also thinking of adding an object like that. but I think it's better to be sure we have more than two parameters before creating it

Reply:
> Also i would not use the term `manager` in the name of the option. I would use something like `--Xnat-kubernetes-pod-name` or `--Xnat-kube-pod-name`

Done. I'm now using `--Xnat-kube-pod-name`